pmedianetwork/webpack-config
# Feature/26s - Expose webpack-plugin-serve

---
Closes #26

---

**How to test (feature)**

I could add it to the setup. Now the setup is simply a highly customizable copy that should work across projects.

**How to test (potential regressions)**
